### DISCUSSION TITLE
Architecture question

### DISCUSSION THREAD

**Discussion by glwallum** [Security-Onion-Solutions/securityonion#4168] [Unsupported Versions]
Hi all,
I am trying to determine the best architecture for out environment with 4 available physical servers. Below is a bit of information for our environment
- High throughput (around 4gpbs)
- 4 servers (all with high memory / disk space)
- Two separate taps (one internal east/west traffic, one external north/south traffic)

Would this be the most appropriate setup?
2 sensor node (fed one tap each)
1 manager/search node
1 search node

Or would this be better?
2 sensor nodes (fed one tap each)
1 manager node
1 search node

Thank you

**Comment by Jaap79** [comment]
Being a fan of separating server roles (which imho is a general best practice anyway) I'd go for option 2: 1 manager, 1 search node. That way you can easily upgrade (or expand the number of) Search nodes later on without messing with the Manager role. It **may** also be beneficial if you want to implement a "True Cluster", not sure if a combined Manager/Search node would be the best way to approach that.

This way you could, technically, also segment your Search Node(s) from the Manager node which reduces the attack surface.

EDIT: I just realised I responded to a question over 1,5 years old.......

**Reply by Acewiza** [reply]
At least you're lookin.  ;-)
